spritz/build.gradle

```diff
@@ -0,0 +1,23 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
```

Comment:
Good to know :D

Reply:
Removed in b2e8c6d2050aaf19b2a02a29d7fdeb4a7573dbb0.